web/modules/custom/apax_search_algolia/react/src/utils/constants.jsx

```diff
@@ -0,0 +1,6 @@
+export const DEBOUNCE_TIME = 300;
+export const LARGE_SCREEN_SIZE = 992; // matches scss variable for lg breakpoint
+
+// Algolia index IDs, from IDs in dashboard
+export const INDEX_ID = 'prod_lexky';
```

Comment:
What's going on here? @JMStewart00

Reply:
This is the id of the index as it is in Algolia. React instantsearch needs to know where to pull the information from.